src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -73,7 +90,11 @@ const SignOrExecuteForm = (props: SignOrExecuteProps): ReactElement => {
 
         <RiskConfirmationError />
 
-        {willExecute ? <ExecuteForm {...props} safeTx={safeTx} /> : <SignForm {...props} safeTx={safeTx} />}
+        {willExecute ? (
+          <ExecuteForm {...props} safeTx={safeTx} />
+        ) : (
+          <SignForm {...props} safeTx={safeTx} isMultisend={isMultisendTx(decodedData)} />
```

Comment:
Should we extend this to check if the tx is a `delegateCall`?
All delegateCall's should not be allowed to batch.